src/components/toolbar-menu/toolbarComponent.ts

```diff
@@ -73,6 +75,36 @@ export class ToolbarController {
   }
 
   /**
+   * Helper method for getting string value of {@link ToolbarType.BUTTON_SELECT}
+   * @memberof ToolbarController
+   * @function getToolbarListType
+   * @returns {string}
+   */
+  public getToolbarListType(): string {
+    return ToolbarType.BUTTON_SELECT.toString();
```

Comment:
honestly, I don't know... enums sound like a great idea but if this is what it takes, I'm starting to doubt the value